internal/shim-sgx/src/handler.rs

```diff
@@ -209,9 +210,9 @@ impl<'a> Handler<'a> {
 
         let c = self.block.cursor();
         let trusted: &[u8] = unsafe { self.aex.gpr.rsi.into_slice(self.aex.gpr.rdx) };
-        let (_, untrusted) = c.copy_slice(trusted).or(Err(libc::EMSGSIZE))?;
+        let (_, untrusted) = c.copy_from_slice(trusted).or(Err(libc::EMSGSIZE))?;
 
-        let req = request!(libc::SYS_write => self.aex.gpr.rdi, untrusted, untrusted.len());
+        let req = request!(libc::SYS_write => self.aex.gpr.rdi, untrusted, trusted.len());
```

Comment:
Similar to the above, why this change?

Reply:
both must be the same length... anyway.. this was a leftover from old code... switched back.